packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -590,7 +600,13 @@ export class AtomicSearchBox {
           this.clearSuggestions();
         }}
         onMouseOver={() => {
-          this.updateActiveDescendant(id);
+          const thisPanel =
```

Comment:
suggestion: Just for readability, this whole method (`renderSuggestions`) is starting to become very compact/dense, with a lot happening on every line.

Might help to at least extract well named private methods, to reduce the complexity when reading/scanning the file quickly.